### ISSUE TITLE
2.4.0 causes OOM errors in docker running node 12.18.2

### ISSUE THREAD

**Issue by radoi90** [webpack-contrib/eslint-webpack-plugin#50]
- Operating System:  Linux Alpine
- Node Version: 12.18.2
- NPM Version:
- webpack Version: 4.44.2
- eslint-webpack-plugin Version: 2.4.0

The project uses an ejected create-react-app base.

### Expected Behavior

Production build to pass.

### Actual Behavior

Production build throws OOM sigkill error.

### Code

```js
new ESLintPlugin({
  // Plugin options
  extensions: ['js', 'mjs', 'jsx', 'ts', 'tsx'],
  formatter: require.resolve('react-dev-utils/eslintFormatter'),
  eslintPath: require.resolve('eslint'),
  context: paths.appSrc,
  cache: true,
  // ESLint class options
  cwd: paths.appPath,
  resolvePluginsRelativeTo: __dirname,
  baseConfig: {
    extends: [require.resolve('eslint-config-react-app/base')],
    rules: {
      ...(!hasJsxRuntime && {
        'react/react-in-jsx-scope': 'error',
      }),
    },
  },
}),
```

**Comment by radoi90** [comment]
Running into the same problem when using the `node:14.15.1-alpine` image too.

**Comment by radoi90** [comment]
Setting `threads: false` seems to solve the issue.

**Comment by evilebottnawi** [comment]
Please create reproducible test repo, we can't fix something somewhere

**Comment by evilebottnawi** [comment]
or provide OOM stack as minimum

**Comment by michaelangeliu** [comment]
As a temp fix, you can try increasing the amount of memory allocated to the node process in the build:
`NODE_OPTIONS=--max_old_space_size=4096`

**Comment by jsg2021** [comment]
Should be adressed in #55

**Comment by Jayaramstarr** [comment]
> Setting `threads: false` seems to solve the issue.

Can you tell me the file and the file path where to set this?
